machine-learning/README.md

```diff
@@ -1,13 +1,15 @@
-# Machine Learning Training Module 
+# Machine Learning Training Module
+
+This CCDL-designed module analyzes pediatric brain tumor data from the [OpenPBTA project](https://github.com/AlexsLemonade/OpenPBTA-analysis) using some machine learning methods.
+OpenPBTA contains data from [Children's Brain Tumor Tissue Consortium](https://cbttc.org/) and the PNOC003 DIPG clinical trial from the [Pediatric Pacific Neuro-oncology Consortium](http://www.pnoc.us/).
+
+This material depends on knowledge gained in the [intro to R](https://github.com/AlexsLemonade/training-modules/tree/master/intro-to-R-tidyverse) module and analyses can be performed within a [Docker container](https://github.com/AlexsLemonade/training-modules/tree/master/docker-install).
```

Comment:
The link to the `Docker container` documentation here appears to be broken.

Reply:
We can link to the Docker repository here. I assume anyone who clicks that link is familiar enough with Docker to know how to get that. https://hub.docker.com/r/ccdl/training_rnaseq